scripts/installNodeModules.sh

```diff
@@ -3,6 +3,6 @@ if [ "$1" = "--production" ]; then
   echo "Installing all node modules"
   rm -rf node_modules | yarn workspaces focus --production
 else
-  echo "Installing all node modules"
+  echo "Installing all node modules from cache"
```

Comment:
Might be worth adding that this first fetches all remote packages to validate cache hasn't been modified with the `--check-cache` flag, in effect installing modules from NPM

Reply:
Good shout, I tried it out locally, and also checked the build logs on the deployment pipeline, so it looks OK to me. (As an aside, apparently even when you use `--check-cache` from the cache, it still fetches from remote https://yarnpkg.com/cli/install#options-check-cache - I didn't know that, but I guess it makes sense)